cmd/mount.go

```diff
@@ -283,6 +350,16 @@ var mountCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
+		if config.IsSet("invalidate-on-sync") {
+			log.Warn("unsupported v1 CLI parameter: invalidate-on-sync is always true in blobfuse2.")
+		}
+		if config.IsSet("pre-mount-validate") {
+			log.Warn("unsupported v1 CLI parameter: pre-mount-validate is always true in blobfuse2.")
+		}
+		if config.IsSet("basic-remount-check") {
```

Comment:
Should add these as well in baseConfig.yaml

Reply:
The point of these is to keep them hidden/only in CLI, so adding them to base config kinda defeats the point there right?

Reply:
Let me know if you still think we need them in baseConfig and I can add it in a follow up PR but I dont think its necessary.